frontend/src/page/RegisterPage.tsx

```diff
@@ -13,21 +14,37 @@ export default function RegisterPage() {
 
     function registerUser(e: FormEvent<HTMLFormElement>) {
         e.preventDefault()
-        axios.post("/api/user/register", {
-            username,
-            password
-        })
-            .then(() => {
-                navigate("/login")
-            })
+        if (password.length > 7) {
```

Comment:
i am not sure if i would work here with such a big if-else-tree. maybe you find a more elegant solutin.